release/release_data_tests.yaml

```diff
@@ -567,6 +567,27 @@
           python setup_chaos.py --chaos TerminateEC2InstanceWithGracePeriod
           --batch-size-to-kill 10 --max-to-kill 100 --kill-delay 120
 
+- name: batch_inference_2
```

Comment:
Planning to later remove `batch_inference_{{scaling}}` and `batch_inference_hetero`, and rename `batch_inference_mock_image_pipeline` to `batch_inference_1`

Reply:
`batch_inference_2` covers the hetero case; `batch_inference_mock_image_pipeline` covers the homogenous (`batch_inference_{{scaling}}`) case.